cmd/sunlight-keygen/keygen.go

```diff
@@ -17,11 +19,15 @@ import (
 )
 
 func main() {
-	if len(os.Args) != 3 {
-		log.Fatal("usage: sunlight-keygen <name> <seed file>")
+	fs := flag.NewFlagSet("keygen", flag.ExitOnError)
+	pemOut := fs.Bool("pem", false, "output keys in PEM format")
```

Comment:
Any reason to retain support for the old format in keygen if we are transitioning off it?

Reply:
Just my instinct for backwards-compatibility and ease of upgrades! But I'm fine with tearing it out since that seems to be your preference, so I've done so in the latest commit. I left the output-switching flag in keygen.go, since the non-PEM format may still be useful for someone (e.g. providing in a log inclusion request).

Reply:
Ah, apologies, I actually meant the other way around: I see how supporting both config options for at least one or two releases helps with migration. I was suggesting removing it only from keygen, since we want to encourage using the new option from now on, and keygen is used when making new logs.

Log inclusion requests can use the new JSON endpoint (and hopefully soon Apple will also take a link to that).

Reply:
Gah, my bad, I misread where you'd left this comment. Done!